File: src/15.ts
import { promises as fs } from 'fs'

const solve15 = async (size: number): Promise<string> => {
    return (await fs.readFile('./res/popular-names.txt', 'utf-8'))
        .split('\n')
        .reverse()
        .slice(0, size)
        .reverse()
        .join('\n')
}

solve15(10).then(console.log)

File: src/15.py
# tail -n 10 res/popular-names.txt

from .helper.file_helper import read_string

source = read_string("./res/popular-names.txt")


def solve15(size: int) -> str:
    return "\n".join(source.split("\n")[-size:])


print(solve15(10))
